docker/mitorsaw/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM quay.io/pacbio/pb_wdl_base@sha256:4b889a1f21a6a7fecf18820613cf610103966a93218de772caba126ab70a8e87
+
+LABEL maintainer="Billy Rowell <wrowell@pacb.com>"
+
+ARG IMAGE_NAME
+ENV IMAGE_NAME "${IMAGE_NAME}"
+ARG IMAGE_TAG
+ENV IMAGE_TAG "${IMAGE_TAG}"
+
+ARG MITORSAW_VERSION
+RUN wget https://github.com/PacificBiosciences/mitorsaw/releases/download/v${MITORSAW_VERSION}/mitorsaw-v${MITORSAW_VERSION}-x86_64-unknown-linux-gnu.tar.gz && \
```

Comment:
Just an FYI: we have a placeholder `--database` option with no actual DB yet. I'll let you know if we actually decide to use that.